packages/pegasus/src/proposals/core-proposal.js

```diff
@@ -40,6 +40,8 @@ export const startPegasus = async ({
     produce: { [CONTRACT_NAME]: produceInstance },
   },
 }) => {
+  [board, namesByAddress] = await Promise.all([board, namesByAddress]);
```

Comment:
non-blocking but I thought you wanted to avoid type changes on the same symbol

Reply:
vestigial.  Thanks for pointing it out.

Oh, this seems to be gone  now.  ?!?

Reply:
still there

Reply:
I kept looking for it in the other core-proposal.  Thanks for pointing it out again.